zingolib/src/lightclient/send.rs

```diff
@@ -23,35 +22,23 @@ impl LightClient {
         ) + 1)
     }
 
+    #[deprecated(note = "superceded by quick_send")]
     /// Send funds
     pub async fn do_send(&self, receivers: Receivers) -> Result<TxId, String> {
-        let transaction_submission_height = self.get_submission_height().await?;
-        // First, get the consensus branch ID
-        debug!("Creating transaction");
-
-        let _lock = self.sync_lock.lock().await;
-        // I am not clear on how long this operation may take, but it's
-        // clearly unnecessary in a send that doesn't include sapling
-        // TODO: Remove from sends that don't include Sapling
-        let (sapling_output, sapling_spend) = crate::wallet::utils::read_sapling_params()?;
-
-        let sapling_prover = LocalTxProver::from_bytes(&sapling_spend, &sapling_output);
-
-        self.wallet
-            .send_to_addresses(
-                sapling_prover,
-                vec![crate::wallet::Pool::Orchard, crate::wallet::Pool::Sapling], // This policy doesn't allow
-                // spend from transparent.
-                receivers,
-                transaction_submission_height,
-                |transaction_bytes| {
-                    crate::grpc_connector::send_transaction(
-                        self.get_server_uri(),
-                        transaction_bytes,
-                    )
-                },
-            )
-            .await
+        let request = match crate::data::receivers::transaction_request_from_receivers(receivers) {
+            Ok(req) => req,
+            Err(e) => {
+                return Err(format!(
+                    "Error: {}\nTry 'help send' for correct usage and examples.",
+                    e
+                ));
+            }
```

Comment:
for what its worth, the correct way of handling our errors moving forward is returning an error struct/enum rather than a string so these "Error: {} yada yadas" are only concocted in the commands.rs / top layer and the Display::fmt implemented on these error types will format the error message